src/ruby/CMakeLists.txt

```diff
@@ -39,29 +39,35 @@ if (RUBY_FOUND)
 
   # Turn on c++
   set_source_files_properties(${swig_i_files} ruby.i PROPERTIES CPLUSPLUS ON)
+  set(SWIG_RB_LIB rbmath)
+  set(SWIG_RB_LIB_OUTPUT math)
```

Comment:
Same here about these Ruby changes, I think they could go in a separate PR

Reply:
https://github.com/ignitionrobotics/ign-math/pull/285